web/src/message_list_view.ts

```diff
@@ -1211,21 +1326,21 @@ export class MessageListView {
         return undefined;
     }
 
-    _new_messages_height(rendered_elems) {
+    _new_messages_height(rendered_elems: JQuery[]): number {
         let new_messages_height = 0;
 
         for (const $elem of rendered_elems.reverse()) {
             // Sometimes there are non-DOM elements in rendered_elems; only
             // try to get the heights of actual trs.
             if ($elem.is("div")) {
-                new_messages_height += $elem.height();
+                new_messages_height += $elem.height()!;
```

Comment:
Perhaps this can be avoided by making `rendered_elems` type more precise. Not sure it's worth the effort, though.

Reply:
I can't think off the top of my head how to do that nicely, since this is working with the generic `JQuery` type.